### ISSUE TITLE
No source text for Commons-android-strings-welcome_help_button_text (commons-android-strings)

### ISSUE THREAD

**Issue by Nikerabbit** [commons-app/apps-android-commons#1772]
**Summary:** 

I am seeing this in our logs:
```No source text for Commons-android-strings-welcome_help_button_text (commons-android-strings)```

It's probably the previously encountered issue about HTML tags present. Could you apply the usual fix?

**Comment by Mansi449** [comment]
@Nikerabbit can you please elaborate upon the issue and also give some insights on how to resolve it?

**Comment by Nikerabbit** [comment]
#1258 has more details.

**Comment by Mansi449** [comment]
Okay! Thanks! I got that. Just wanted to know at what point does `No source text for Commons-android-strings-welcome_help_button_text (commons-android-strings)` get logged?

**Comment by Nikerabbit** [comment]
It's a sanity check I've put in our (translatewiki.net) code to catch issues like this.

**Comment by misaochan** [comment]
This looks to be similar to #1333 . I will copy my suggestion here:

So, what needs to be done in this task is:

1. go to strings.xml and search for all strings that have a <u> tag added but no CDATA tag
2. add the CDATA tag to those strings

**Comment by misaochan** [comment]
Note to collaborators @maskaravivek @neslihanturan @nicolas-raoul  - In the future when we merge PRs, we should check that the strings added do not have HTML tags without CDATA.

**Comment by Mansi449** [comment]
Thanks @Nikerabbit @misaochan for the clarification.
I would like to work on this issue.

**Comment by misaochan** [comment]
@Mansi449 wonderful! However, it seems that both you and @arshadkazmi42 have expressed interest simultaneously in working on this in separate issues, so I am not sure who to assign it to. Could you both please discuss among yourselves and hopefully come to an agreement? :) Thanks!

**Comment by arshadkazmi42** [comment]
@Mansi449 if you have started working on this you can take this up. If you would like me to work on it, let me know.

**Comment by Mansi449** [comment]
Thanks a lot @arshadkazmi42 for letting me take this up.

**Comment by misaochan** [comment]
Thanks guys! @Mansi449 it is yours. :)

**Comment by Mansi449** [comment]
I added the CDATA tag to a string:
`  <string name="welcome_help_button_text"><![CDATA[<u>More Information</u>]]></string>
`
But it gives the following result:
![screenshot_20180813-175601](https://user-images.githubusercontent.com/32809297/44222911-b39dbb00-a1a3-11e8-93b5-e62d4763420e.png)

I think something like this has to be done:
`TextView tv = findViewById(R.id.welcomeInfo);` (welcomeInfo is the TextView that contains the string "More Information").
Then:
`tv.setText(Html.fromHtml(String.valueOf(R.string.welcome_help_button_text)));`

@Nikerabbit @misaochan What do you think?

**Comment by maskaravivek** [comment]
Yes, you are right. You need to use:

```
tv.setText(Html.fromHtml(String.valueOf(R.string.welcome_help_button_text)));
```

**Comment by Mansi449** [comment]
Thanks @maskaravivek I would make the necessary changes.

**Comment by Mansi449** [comment]
@Nikerabbit @maskaravivek @misaochan I have made the required changes for the `welcome_help_button_text` string. 
But another string `<string name="nominated_see_more"><u>See webpage for details</u></string>` has the same issue.
So shall I create a PR for this issue now and fix the `nominated_see_more` string in a separate issue or shall I create the PR only after fixing the `nominated_see_more` string also?

**Comment by maskaravivek** [comment]
IMO it should be fine to fix both the strings in a single PR. You can name the PR accordingly.

**Comment by Mansi449** [comment]
Ok sure, thanks!
